### PR TITLE
Search order

### PR DESCRIPTION
**Pull request by MelissaLaskowski**

### CODE REVIEW

**Comment by corbanmailloux** [comment]
@Daniel-J-Fisher Before you leave for "vacation," could you do a review of this? I've approved it, but I made many of my own changes so I want another dev. to review it.

**Review by corbanmailloux — APPROVED** [review]

**Review by Daniel-J-Fisher — APPROVED** [review]
The big modal looks excellent and everything is functioning correctly. It's quite cool to see all of the orders coming back.
In a nice-to-have design note: I really don't like how our application tends to dump information into a select box and does not allow a way to actually open up that information in a more detailed view. It's not something I'm going to deny this PR for because it happens in multiple places in the application and it's sort of our style now.

tldr: If we have time/care enough, we should consider updating our design for selecting options.
Approving PR.

**Comment by corbanmailloux** [comment]
Agreed about the select box thing. It's been bothering me too, but I figured we should get this in first so we can use the system and then clean it up after. 
I'm thinking we could use something like the table structure in the new ticket screen.
